aclk/agent_cloud_link.c

```diff
@@ -724,8 +730,10 @@ void aclk_del_collector(const char *hostname, const char *plugin_name, const cha
 
     if (unlikely(agent_state == AGENT_INITIALIZING))
         last_init_sequence = now_realtime_sec();
-    else
-        aclk_queue_query("on_connect", NULL, NULL, NULL, 0, 1, ACLK_CMD_ONCONNECT);
+    else {
+        if (unlikely(aclk_queue_query("connector", NULL, NULL, NULL, 0, 1, ACLK_CMD_ONCONNECT)))
```

Comment:
@stelfrag This is the change I mentioned on slack

Reply:
Yes that is fine. The first parameter (topic) is not used with non cloud commands but required. This is to better show that its called from a collector not **connector** :-1: 

I will adjust the names

Reply:
Parameter fixed to collector (also function calls that had "connector" instead of collector)